sdks/python/apache_beam/transforms/sideinputs_test.py

```diff
@@ -309,6 +316,65 @@ def test_flattened_side_input(self):
     assert_that(results, equal_to(['a', 'b']))
     pipeline.run()
 
+  @attr('ValidatesRunner')
+  def test_multi_triggered_gbk_side_input(self):
+    """Test a GBK sideinput, with multiple triggering."""
+    options = PipelineOptions()
+    options.view_as(StandardOptions).streaming = True
+    p = TestPipeline(options=options)
+
+    main_stream = (p
+                   | 'main TestStream' >> TestStream()
+                   .advance_watermark_to(3)
+                   .add_elements(['a1'])
+                   .advance_watermark_to(8)
+                   .add_elements(['a2'])
+                   | 'main windowInto' >> beam.WindowInto(
+                       window.FixedWindows(5),
+                       trigger=trigger.AfterWatermark(
+                           early=trigger.AfterCount(1)),
+                       accumulation_mode=trigger.AccumulationMode.DISCARDING))
+
+    emit_vals = Map(lambda k_vs: k_vs[1])
+
+    side_stream = (p
+                   | 'side TestStream' >> TestStream()
```

Comment:
Multiple test streams in the same pipeline have undefined behavior. What you should do is create a single test stream with both and then partition them into the main and side inputs.

Reply:
fixed by using test stream's multiple output.